tests/commands/test_commands.py

```diff
@@ -409,3 +409,39 @@ def mock_write(tx, mode):
     resp = run_upsert_validator_new(args, b)
 
     assert b.get_transaction(resp)
+
+
+@pytest.mark.tendermint
+@pytest.mark.abci
```

Comment:
This is not the intended usage of the `abci` marker i.e. a test which uses the `abci` marker doesn't need to mock `write_transaction` api as Tendermint is running during tests and should be used to posting transactions.

Reply:
Resolved